README.md

```diff
@@ -43,9 +43,10 @@ Maps to `docker unpause`.
 
 ### `provision`
 Either calls Docker's `build` or `pull`, depending on whether a Dockerfile is specified. The Docker cache can be disabled by passing `--no-cache`.
+A specific tag can be build using the `--tag` argument;
 
 ### `push`
-Maps to `docker push`.
+Maps to `docker push`. By using the `--tag` parameter, you can push your image with a specific tag, instead of the version that is in your Crane config file.
```

Comment:
Maybe more precise would be `instead of the version that may be specified in your config file`.